### PR TITLE
Minor documentation updates for validators

### PR DESCRIPTION
**Pull request by unordered-set**
## Motivation

Fix documentation for validators ASAP

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

(You must have submitted a [signed CLA](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#contributor-license-agreement) that includes your GitHub handle prior to us accepting and landing your work. Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site, and link to your PR here.)

### CODE REVIEW

File: developer-docs-site/docs/tutorials/validator-node/using-source-code.md [hunk 1]
```diff
@@ -44,31 +44,31 @@ With your development environment ready, now you can start to setup your Validat
 7. Generate key pairs (node owner key, consensus key and networking key) in your working directory.
 
     ```
-    cargo run -p aptos -- genesis generate-keys --output-dir ~/$WORKSPACE
+    cargo run --release -p aptos -- genesis generate-keys --output-dir ~/$WORKSPACE
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/tutorials/validator-node/using-source-code.md:47]: I think if `--release` is omitted it builds a debug binary, which is slower but still a valid one.

Comment (rajkaramchedu) [reply developer-docs-site/docs/tutorials/validator-node/using-source-code.md:47]: See the NOTE below step 9 here in this doc:https://aptos.dev/tutorials/full-node/run-a-fullnode/#using-aptos-core-source-code

**Review by rajkaramchedu — APPROVED** [review]

File: developer-docs-site/docs/tutorials/validator-node/using-source-code.md (hunk 1 above)
Comment (unordered-set) [reply developer-docs-site/docs/tutorials/validator-node/using-source-code.md:47]: Yes, it is true, my point here was that the final build for the node actually done in release. But having both release and debug builds is a waste of time and disk absolutely for nothing

**Comment by unordered-set** [comment]
Thanks for your time!

**Pull request closed without merging**